source/server/getting-started/migrating.rst

```diff
@@ -29,19 +32,42 @@ Worlds
 
 Forge, and thus SpongeForge (and also SpongeVanilla), use the same world structure as vanilla Minecraft. Vanilla Minecraft
 places the nether (typically ``world_nether``) and the end (typically ``world_the_end``) dimensions within the
-``world`` folder.
+``world`` folder. However Bukkit and Spigot *don't* use this system to save the worlds, thus migration is needed.
+
+SpongeForge and SpongeVanilla provide a fully automated conversion script which converts your worlds for you.
+This is how it works:
+
+1. Shutdown your Bukkit or Spigot server and Backup the entire folder.
+
+#. Place SpongeForge or SpongeVanilla in the folder where you ran your old server, remove the Bukkit or Spigot jars.
 
-CraftBukkit relocates the nether and end dimensions outside of the ``world`` folder, which must be remedied if it is
-desired to retain the nether and end dimensions when running SpongeForge. To relocate the nether and end dimensions, follow
-the instructions below.
+#. Start the Sponge server, the migrator will be loaded automatically.
 
-.. tip::
+#. The Migrator will look into the ``bukkit.yml`` for a config key called ``world-container`` and will search that folder for
+   worlds to transfer. If that file isn't around (or something caused it to fail to read), then the migrator will use
+   the ``root`` folder of the server (which is CraftBukkit standard).
 
-    It is a good idea to make a backup of your worlds in case you make a mistake.
+#. Now the migration is performed. At this point, worlds are moved over from what we call the ``world container`` into
+   the folder defined in the :doc:`configuration/server-properties` file via the ``level-name`` key.
+
+#. Bukkit puts things in weird places (with weird names), thus two important fixes have to be applied. Keep in mind that
+   these fixes are based on *assumptions* (due to Bukkit's structure).
+
+   * The first fix is to rename any folder whose name starts with ``level-name`` property and ends with a Vanilla
+     dimension name (``_nether``/``_the_end``) to ``DIM-1`` and ``DIM1`` respectively.
+
+   * The second fix is to migrate up the ``region`` data within a Bukkit Vanilla nether/the_end. Bukkit puts this data
+     into ``DIM-1\region`` and ``DIM1\region`` respectively whereas Vanilla/Forge expect ``region`` to be in the
+     **root** of the world's folder structure.
+
+The migrator can't provide all needed configuration values. This is the reason why you need to change several parameters
+by hand to make the world work properly upon loading it on SpongeForge or SpongeVanilla. Usually a
```

Comment:
Changed wording here, i guess it's best to just encourage users to use a world management plugin. Will hopefully save us headaches in the future.